app/src/main/java/com/example/composetemplate/data/remote/requests/FirebaseUserRequests.kt

```diff
@@ -14,7 +15,8 @@ sealed class FirebaseUserRequests {
         override val headers: MutableMap<String, String>? = null,
         override val body: Any?,
         override val timeout: Long? = null,
-        override val baseUrl: String = Constants.FIREBASE_BASE_URL,
+        override val baseUrl: String = FirebaseConfigProvider.getData(FirebaseConfigProvider.RemoteConfigValues. BASE_FIREBASE_URL),
```

Comment:
I know its a little bit niggling but can you do formatting? maybe make it FirebaseConfigProvider.RemoteConfigValues to RemoteConfigValues using typealias

Reply:
Ok good idea